ftplugin/colortemplate.vim

```diff
@@ -11,10 +11,8 @@ let b:did_ftplugin = 1
 let s:undo_ftplugin = "setlocal commentstring< omnifunc< | unlet! b:colortemplate_outdir"
 let b:undo_ftplugin = (exists('b:undo_ftplugin') ? b:undo_ftplugin .. '|' : '') .. s:undo_ftplugin
 
-let b:colortemplate_outdir = empty(expand('%:p:h')) ? getcwd() : expand('%:p:h')
-if b:colortemplate_outdir =~? '\m\%(color\)\=templates\=$'
-  let b:colortemplate_outdir = fnamemodify(b:colortemplate_outdir, ':h')
-endif
```

Comment:
This conditional should not be removed. When a template is inside a directory called `template[s]` or `colortemplate[s]`, which is the convention, the output directory should be set one level up, so that the `colors` directory is generated at the right location, and not inside the template's directory.